ducktape/tests/runner.py

```diff
@@ -58,9 +61,15 @@ def __init__(self, *args, **kwargs):
         self.current_test = None
         self.current_test_context = None
 
+        # Move regression tests to the end of the queue
+        regression_tests = [t for t in self.tests if isinstance(t.cls, RegressionTest)]
```

Comment:
Long term, I would like to avoid customized support in the runner like this. That said, I think since this is completely internal right now, we can clean this up later. We'll eventually want to be able to express test dependencies, and when we can, the test loader can setup the dependency so that the test runner will handle it generically, rather than this customized approach for `RegressionTest` we're using now. Might want to file an issue so we don't lose track of this.